docs/source/install/docker.rst

```diff
@@ -127,3 +127,14 @@ To stop the containers, run:
 
   docker-compose down
 
+To run from the command line inside the container:
+
+.. sourcecode::bash
```

Comment:
Looks like this block is not rendered well in rst: https://github.com/StackStorm/st2docs/blob/c6025ba2152bddcac747adb7b8a540c5bcb83797/docs/source/install/docker.rst

There seems to be formatting issue here:

```suggestion
.. sourcecode:: bash
```